core/dbt/cli/params.py

```diff
@@ -117,46 +117,46 @@
 log_format = click.option(
     "--log-format",
     envvar="DBT_LOG_FORMAT",
-    help="Specify the log format, overriding the command's default.",
+    help="Specify the format of logging to the console and the log file. Use --log-format-file to configure the format for the log file differently than the console.",
```

Comment:
I added a quite a bit of verbosity in this commit (https://github.com/dbt-labs/dbt-core/pull/7176/commits/3097cf3cdce093c6e014cb2f465f1732a0758a23).

I'd love your eyes on these changes since you originally added this functionality in [#6994](https://github.com/dbt-labs/dbt-core/pull/6994).

The new behavior is logical & consistent across 3 different flags, but it wasn't immediately intuitive for me. The reason I added so much additional explanation is because I found it a bit tricky to understand how the different options behaved.

Would you be willing to give this a peek and let me know if the changes are "okay" :white_check_mark:  or "needs improvement" :x: ?